### PR TITLE
Update advanced-features.md

### PR DESCRIPTION
**Pull request by edberei**
Made clarification to documentation

### CODE REVIEW

File: microsoft-365/security/defender-endpoint/advanced-features.md [hunk 1]
```diff
@@ -43,6 +43,9 @@ Turn on this feature to take advantage of the automated investigation and remedi
 
 ## Live response
 
+> [!NOTE]
+> Live Response requires Automated investigation to be turned on before you can enable it in the advanced settings section in the MDE portal.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/advanced-features.md]:
```suggestion
> Live response requires **Automated investigation** to be turned on before you can enable it in the advanced settings section in the Microsoft for Endpoint portal.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @edberei, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@edberei Gentle reminder. Thanks

**Comment by edberei** [comment]
I believe I just pushed it out, thank you!

Ed Berei
Support Engineer | Endpoint Protection
Customer Service & Support | Microsoft Corporation
*(980) 776-1263  * ***@***.******@***.***>
Working Hours: Mon-Fri 8:30 am - 5:30pm EST (UTC-4)
If you'd like to provide feedback about my service, please e-mail my manager at ***@***.******@***.***>

***@***.***

**Pull request merged**